MatthewKelly12/here_we_grow_django_py
# added requirements.txt

## Link to Ticket
Closes
 
## Description of Proposed Changes
-added requirements.txt


## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout requirements
python3 -m venv <name of virtual environment>
pip install -r requirements.txt
```

1. verify requirements.txt


## Impacted Areas in Application

-
